package.json

```diff
@@ -8,20 +8,23 @@
   "private": true,
   "scripts": {
     "build": "webpack",
-    "start": "webpack-dev-server --hot --color --progress --info=true"
+    "start": "webpack-dev-server --hot --color --progress --info=true --config config/webpack.config.js"
```

Comment:
Similar request here, could we move the webpack config back to the root directory temporarily - just for the sake of making this PR only reflect the minimal changes required to get the success notification working?

Of course, we do want this though. Similar to prettier support request, you could post this as a separate PR toward #7 ?